interface/src/main.cpp

```diff
@@ -66,59 +66,215 @@ int main(int argc, const char* argv[]) {
 
     setupHifiApplication(BuildInfo::INTERFACE_NAME);
 
-    QStringList arguments;
-    for (int i = 0; i < argc; ++i) {
-        arguments << argv[i];
-    }
-
     QCommandLineParser parser;
     parser.setApplicationDescription("Vircadia");
-    QCommandLineOption versionOption = parser.addVersionOption();
     QCommandLineOption helpOption = parser.addHelpOption();
+    QCommandLineOption versionOption = parser.addVersionOption();
 
-    QCommandLineOption urlOption("url", "", "value");
-    QCommandLineOption noLauncherOption("no-launcher", "Do not execute the launcher");
-    QCommandLineOption noUpdaterOption("no-updater", "Do not show auto-updater");
-    QCommandLineOption checkMinSpecOption("checkMinSpec", "Check if machine meets minimum specifications");
-    QCommandLineOption runServerOption("runServer", "Whether to run the server");
-    QCommandLineOption serverContentPathOption("serverContentPath", "Where to find server content <path>", "serverContentPath");
-    QCommandLineOption allowMultipleInstancesOption("allowMultipleInstances", "Allow multiple instances to run");
-    QCommandLineOption overrideAppLocalDataPathOption("cache", "set test cache <dir>", "dir");
-    QCommandLineOption overrideScriptsPathOption(SCRIPTS_SWITCH, "set scripts <path>", "path");
-    QCommandLineOption responseTokensOption("tokens", "set response tokens <json>", "json");
-    QCommandLineOption displayNameOption("displayName", "set user display name <string>", "string");
-    QCommandLineOption setBookmarkOption("setBookmark", "set bookmark key=value pair", "string");
-    QCommandLineOption defaultScriptOverrideOption("defaultScriptsOverride", "override defaultsScripts.js", "string");
-    QCommandLineOption forceCrashReportingOption("forceCrashReporting", "Force crash reporting to initialize");
+    QCommandLineOption urlOption(
+        "url",
+        "Start at specified URL location.",
+        "string"
+    );
+    QCommandLineOption protocolVersionOption(
+        "protocolVersion",
+        "Writes the protocol version base64 signature to a file?",
+        "path" // Why??
+    );
+    QCommandLineOption noUpdaterOption(
+        "no-updater",
+        "Do not show auto-updater."
+    );
+    QCommandLineOption checkMinSpecOption(
+        "checkMinSpec",
+        "Check if machine meets minimum specifications. The program will run if check passes."
+    );
+    QCommandLineOption runServerOption(
+        "runServer",
+        "Run the server."
+    );
+    QCommandLineOption listenPortOption(
+        "listenPort",
```

Comment:
This is missing a value name, so the option is described as:
```
  --listenPort                             Port to listen on.
```

Note the missing value name, eg:
```
  --url <string>                           Start at specified URL location
```

```